src/descriptor/template.rs

```diff
@@ -443,6 +444,9 @@ mod test {
         is_fixed: bool,
         expected: &[&str],
     ) {
+        let secp = Secp256k1::new();
+        let deriv_ctx = DescriptorPublicKeyCtx::new(&secp, ChildNumber::Normal { index: 0 });
```

Comment:
Everywhere else we use `ChildNumber::from_normal_idx(0)`. Let's be consistent and use that here, too.